engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -677,4 +697,29 @@ public RenderingStage getCurrentRenderStage() {
     public void recompileShaders() {
         shaderManager.recompileAllShaders();
     }
+
+    @Override
+    public void initialise() { }
+
+    @Override
+    public void preBegin() { }
+
+    @Override
+    public void postBegin() { }
+
+    @Override
+    public void preSave() { }
+
+    @Override
+    public void postSave() { }
+
+    @Override
+    public void shutdown() { }
+
+    @Command(shortDescription = "Debugging command for DAG.", requiredPermission = PermissionManager.NO_PERMISSION)
+    public void dagNodeCommand(@CommandParam("nodeUri") final String nodeUri, @CommandParam("command") final String command, @CommandParam("arg1") final String arg1) {
+        // TODO: Convert arg1 to args[]
```

Comment:
Actually, I was *really* confused by the existing javadocs of `@Command`, and hence put this off for later.

I guess just like all of us, they also need some love :D